codegen.ts

```diff
@@ -233,6 +233,10 @@ const config: CodegenConfig = {
           },
         },
         scalars: {
+          ID: {
```

Comment:
Should we update the client file as well to ensure it matches both frontend and backend?